packages/cli/src/databases/repositories/execution.repository.ts

```diff
@@ -565,7 +567,9 @@ export class ExecutionRepository extends Repository<ExecutionEntity> {
 			return;
 		}
 
-		await this.binaryDataService.deleteMany(workflowIdsAndExecutionIds);
```

Comment:
Do we still need the `deleteMany` in `ObjectStoreManager` ? or should we make it optional, and implement that method only in `FileSystemManager`.

Reply:
If we manually delete an execution, we'll need to delete all binary data items: https://github.com/n8n-io/n8n/pull/7469/files#r1366681929

Reply:
do we really need to do that? why not let the lifecycle policy take care of that as well?
then we have less code to maintain.